src/lammpsio/snapshot.py

```diff
@@ -176,19 +183,90 @@ def to_hoomd_gsd(self, type_map=None):
             frame.particles.mass = self.mass.copy()
         if self.has_molecule():
             frame.particles.body = self.molecule - 1
-
+        if self.has_bonds():
+            frame.bonds.N = self.bonds.N
+            frame.bonds.types = self.bonds.id
+            frame.bonds.group = self.bonds.members - 1
```

Comment:
Is this acceptable to just subtract one to move from LAMMPS one-index and HOOMD's zero-index?  I couldn't think of any scenarios where this wouldn't work, but wanted to check.

Reply:
It should be, this would be how we implicitly shift the `id`.